lib/fathead/mdn_css/output.txt

```diff
@@ -344,28 +389,151 @@
rsed as a CSS <angle> dimension, that is including the unit (e.g. 30.5deg). If it is not valid, that is not an angle or out of the range accepted by the CSS property, the default value is used.\n     Leading and trailing spaces are stripped.\n    0deg, or, if 0deg is not a valid value for the property, the property's minimum value.\n   \n   \n    deg, grad, rad  \n    <angle>\n    The attribute value is parsed as a CSS <number>, that is without the unit (e.g. 12.5), and interpreted as an <angle> with the specified unit. If it is not valid, that is not a number or out of the range accepted by the CSS property, the default value is used.\n     Leading and trailing spaces are stripped.\n    0deg, or, if 0deg is not a valid value for the property, the property's minimum value.\n   \n   \n    time  \n    <time>\n    The attribute value is parsed as a CSS <time> dimension, that is including the unit (e.g. 30.5ms). If it is not valid, that is not a time or out of the range accepted by the CSS property, the default value is used.\n     Leading and trailing spaces are stripped.\n    0s, or, if 0s is not a valid value for the property, the property's minimum value.\n   \n   \n    s, ms  \n    <time>\n    The attribute value is parsed as a CSS <number>, that is without the unit (e.g. 12.5), and interpreted as an<time> with the specified unit. If it is not valid, that is not a number or out of the range accepted by the CSS property, the default value is used.\n     Leading and trailing spaces are stripped.\n    0s, or, if 0s is not a valid value for the property, the property's minimum value.\n   \n   \n    frequency  \n    <frequency>\n    The attribute value is parsed as a CSS <frequency> dimension, that is including the unit (e.g. 30.5kHz). If it is not valid, that is not a frequency or out of the range accepted by the CSS property, the default value is used.\n    0Hz, or, if 0Hz is not a valid value for the property, the property's minimum value.\n   \n   \n    Hz, kHz  \n    <frequency>\n    The attribute value is parsed as a CSS <number>, that is without the unit (e.g. 12.5), and interpreted as a <frequency> with the specified unit. If it is not valid, that is not a number or out of the range accepted by the CSS property, the default value is used.\n     Leading and trailing spaces are stripped.\n    0Hz, or, if 0Hz is not a valid value for the property, the property's minimum value.\n   \n   \n    %  \n    <percentage>\n    The attribute value is parsed as a CSS <number>, that is without the unit (e.g. 12.5), and interpreted as a <percentage>. If it is not valid, that is not a number or out of the range accepted by the CSS property, the default value is used.\n     If the given value is used as a length, attr()computes it to an absolute length.\n     Leading and trailing spaces are stripped.\n    0%, or, if 0% is not a valid value for the property, the property's minimum value.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/attr#Values
```

Comment:
I see that we're capturing a table and converting it to text, which is causing all the extra newlines. We might actually be able to keep the table HTML if we can strip out `<a>` from it...